src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -709,6 +679,19 @@ void CC32xxWiFi::run_on_network_thread(std::function<void()> callback)
 }
 
 /*
+ * CC32xxWiFi::fd_remove()
+ */
+void CC32xxWiFi::fd_remove(int16_t socket)
```

Comment:
please rename the argument from socket to sd
this is to avoid confusion of different address spaces: fd is in our abstraction, but sd is in the simplelink abstraction.
`socket` is not sufficiently specific, because bind(), listen() etc have arguments called `socket` but they are in fact in our fd array.

Consider changing the arguments to other functions similar to this (e.g. fd_set_read, add_socket, del_socket, etc)